buf.yaml

```diff
@@ -21,6 +21,7 @@ lint:
     - DEFAULT
 breaking:
   ignore:
+    - state/queues/v1/queues.proto
```

Comment:
I think instead of adding this ignore, you could also add the `CI_OVERRIDE_BUF_BREAKING` label to this PR to skip the buf check.

Reply:
Ack. But I'm removing this right after this is merged anyway.